docs/user-guide/ze-profiles.md

```diff
@@ -42,94 +153,37 @@ Follow these steps:
-
-Extensions built on Zowe Explorer provide users with additional functionality through unique profile types, such as RSE or FTP. The "associate profiles" function allows users to link, or associate, these extension profile types with an existing zOSMF profile. Every Zowe Explorer profile has two types of association: primary and secondary. While the primary association is `zosmf`, the secondary association includes Zowe CLI plug-ins or other services you might have. Within the scope of their own extender package, associated profiles can be used to access a relevant REST API that is available to the extender.
-
-Create a secondary association for your Zowe Explorer profiles.
-
-**Follow these steps:**
-
-1. Navigate to the explorer tree.
-2. Right-click your profile
-3. Select the **Associate profiles** option.
-4. Select a secondary profile type.
-5. Select a file name you want to create an association for.
-
-<img src={require("../images/ze/ZE-associate.gif").default} width="600" height="300" alt="Edit a Profile"/>
+## Using base profiles and tokens with existing profiles
 
-You have successfully created the secondary association for your Zowe Explorer profile.
+As a Zowe user, you can leverage the base profile functionality to access multiple services through Single Sign-on. Base profiles enable you to authenticate using Zowe API Mediation Layer (API ML). You can use base profiles with more than one service profile. For more information, see [Base Profiles](../user-guide/cli-using-using-profiles.md#base-profiles).
 
-For more information, see [Associating Zowe CLI Profiles](https://github.com/zowe/vscode-extension-for-zowe/blob/master/docs/README-Extending.md#associating-zowe-cli-profiles).
+Before you log in and connect your service profile, ensure that you have [Zowe CLI](../user-guide/cli-install-cli-checklist.md) v6.16 or higher installed.
 
-## Enabling Secure Credential Store with Zowe Explorer
+### Accessing services through API ML using SSO
 
-Store your credentials securely by using the Secure Credential Store (SCS) plug-in in Zowe Explorer. By default, your credentials are stored in plain text.
-
-Activate the SCS plug-in in Zowe Explorer.
+Connect your service profile with a base profile and token.
 
 **Follow these steps:**
 
-1. Open Zowe Explorer.
-2. Navigate to the VSCode settings.
-3. Open Zowe Explorer Settings.
-4. Add the **Zowe-Plugin** value to the `Zowe Security: Credential Key` entry field.
-5. Restart VSCode.
-6. Create a profile.
-
-Your Zowe Explorer credentials are now stored securely.
-
-### For Zowe CLI users
-
-Ensure that you install the SCS plug-in for Zowe CLI before activating SCS in Zowe Explorer. For more information about the SCS plug-in for Zowe CLI, see [Secure Credential Store plug-in for Zowe Explorer](cli-scsplugin.md).
+1. Open Zowe CLI and run the following command:
```

Comment:
Are we trying to illustrate that people can use tokens between CLI and ZE ?
or do just that ZE works with tokens ?

If we only care about SSO through APIML (ZE using tokens) we should probably remove the CLI references from here
(in the future)

Reply:
ummm - i see only one ref to CLI in the topic.  Open CLI..... i'm calling this done.